configure/kubectl-apply-all.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# This file should contain `kubectl apply` commands for all of your configured resources.
+# 
+# This file should be run:
+#   * When the cluster is first created
+#   * Whenever the configuration for any resource has been updated
+
+kubectl apply --prune -l deploy=sourcegraph -f base --recursive
```

Comment:
we also need to make sure this command is run from the top-level repo dir. I think this script should probably live top-level directory given it is doing something to base (and in the future to configure, etc)